gusto/linear_solvers.py

```diff
@@ -156,14 +159,26 @@ def V(u):
         dxp = dx(degree=(self.quadrature_degree))
         dS_vp = dS_v(degree=(self.quadrature_degree))
 
+        # introduce density potential temperatures
+        theta_rho = theta
```

Comment:
`theta_rho` is the "density potential temperature", θ_ρ defined as θ_v / (1 + total_water). When there is no water, this is just the same as the dry θ. The name is definitely confusing, I don't know what else to call it!